### DISCUSSION TITLE
How to use .content-visible with headers?

### DISCUSSION THREAD

**Discussion by alexCardazzi** [quarto-dev/quarto-cli#6257] [Q&A] [answered]
### Description

I am trying to use the same file to render both html and revealjs output formats.  So far, I have been successful.  The only issue is hiding headers when rendering html vs displaying them when rendering revealjs.  I want my slides to have headers but I do not want redundant headers for the html output.  Currently, I am using the following:

````qmd
## Some Slide Title

Example Notes 1

::: {.content-visible when-format="revealjs"}
## Some Slide Title
:::

Example Notes 2
````

Rendering this results in the desired behavior in html but not revealjs.  The revealjs version contains "Example Notes 2" at the top left of every single slide.  Is there something where I can remove consecutive headers that are exactly the same (for the html, but not revealjs)?  Any thoughts would be appreciated.

**Comment by mcanouil** [comment]
I don't understand your issue (see below).

Could you share a complete and reproducible example exhibiting the issue, and possibly with screenshots?

<details><summary>Working example</summary>

````qmd
---
title: "Quarto Playground"
format:
  revealjs: default
  html: default
---

## Some Slide Title

Example Notes 1

::: {.content-visible when-format="revealjs"}
## Some Slide Title
:::

Example Notes 2
````

</details> 

| revealjs | html |
|--------|--------|
| <img width="727" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/2afe3f86-aaca-4759-9103-2c3eb332b246"> | <img width="727" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/e56fc954-d703-4953-8cf0-016f2bb45f3f"> |

**Reply by alexCardazzi** [reply]
Thanks for this.  Using the same code you supplied, this is what I was able to generate.
![image](https://github.com/quarto-dev/quarto-cli/assets/47037506/275caf71-b8b8-41ee-8576-65d1a8890067)
![image](https://github.com/quarto-dev/quarto-cli/assets/47037506/4f4e9e32-edef-46d3-988d-6eb1914a3890)

**Reply by mcanouil** [reply, marked as answer]
What version of Quarto are you using (output of `quarto check`) and using which browser?

**Reply by alexCardazzi** [reply]
Good question... Apparently, I needed up download a newer version.  Thanks for the help!
